### PR TITLE
Added letters for automation scenarios

### PR DESCRIPTION
**Pull request by zouxuoz**
Added ability to create, edit and delete letters for using with user's automation scenarios.

### CODE REVIEW

File: src/resources/template/letter/model.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        },
+
+        parseFromPreset: function(preset) {
+            this.set('id', preset.id);
+            this.set('name', i18n(preset.id));
+            this.set('subject', i18n(preset.id));
+        },
+
+        serialize: function() {
```
Comment (Radiansz) [inline src/resources/template/letter/model.js]: It's better to call parent serialize function to avoid duplicate code

**Comment by zouxuoz** [comment]
@Radiansz fixed serialize function

File: src/pages/automation/letters/wizard/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        collections: {
+            templates: function() {
+                return new TemplateCollection({
+                    fetchModels: true
+                });
+            }
+        },
+
+        events: {
+            'click .js-save-wizard': function(event) {
```
Comment (Radiansz) [inline src/pages/automation/letters/wizard/index.js]: Empty string after each string with code seems little weard

**Comment by Radiansz** [comment]
lgtm

**Pull request merged**